modules/oxide/src/less/theme/components/menubar/promotion.less

```diff
@@ -0,0 +1,46 @@
+//
+// Promotion
+//
+
+@promotion-background-color: #E8F1F8;
+@promotion-background-hover-color: #B4D7FF;
+@promotion-background-active-color: #D9EDF7;
+@promotion-color: #086BE6;
+
+// private variables
+@_promotion-row-separator-color: escape('@{menubar-row-separator-color}');
+@_promotion-height: @_promotion-select-height + (@_promotion-select-spacing-y * 2) + 1px;
+@_promotion-divider-height: @_promotion-height - 1px;
+@_promotion-select-height: @menubar-select-height;
+@_promotion-select-spacing-y: @menubar-select-spacing-y;
```

Comment:
I know I commented about this previously, but these definitely aren't used anymore so we can remove them.

```suggestion
```

Reply:
Right, I forgot to remove them when I removed the background. That's entirely on me.